### PR TITLE
Various style tweaks

### PR DESCRIPTION
**Pull request by benhaynes**

### CODE REVIEW

File: src/components/v-pagination/v-pagination.vue [hunk 1]
```diff
@@ -136,41 +127,38 @@ export default defineComponent({
 .v-pagination {
 	--v-pagination-active-color: var(--primary);
 
+	.gap {
+		margin: 0 4px;
+		color: var(--background-normal-alt);
```
Comment (rijkvanzanten) [inline src/components/v-pagination/v-pagination.vue:132]: Background color for foreground content 👀 

Shouldn't this be foreground-subdued instead?

File: src/lang/en-US/index.json [hunk 2]
```diff
@@ -240,7 +240,7 @@
 	"upload_file_indeterminate": "Uploading File | Uploading files {done}/{total}",
 	"upload_file_success": "File Uploaded | {count} Files Uploaded",
 	"upload_file_failed": "Couldn't Upload File | Couldn't Upload Files",
-	"view_type": "View As...",
+	"view_type": "Layout Type",
```
Comment (rijkvanzanten) [inline src/lang/en-US/index.json:243]: The translation key is still `view_type`, so the translation should match that ->  `View Type`.

Lets change `view_type` to `layout_type` and use that new translation in layout picker drawer detail

File: src/layouts/tabular/tabular.vue [hunk 3]
```diff
@@ -78,14 +69,26 @@
...
+							:length="totalPages"
+							:total-visible="7"
+							show-first-last
+							:value="page"
+							@input="toPage"
+						/>
+					</div>
+
+					<div class="per-page">
+						<span>{{ $t('layouts.tabular.per_page') }}</span>
```
Comment (rijkvanzanten) [inline src/layouts/tabular/tabular.vue:85]: Per page is so generic, lets use the translation from `index.json` instead

```suggestion
						<span>{{ $t('per_page') }}</span>
```

File: src/components/v-pagination/v-pagination.vue (hunk 1 above)
Comment (benhaynes) [reply src/components/v-pagination/v-pagination.vue:132]: It was too light (couldn't see it)... but we can change if needed.

Comment (benhaynes) [reply src/components/v-pagination/v-pagination.vue:132]: Or maybe it was too dark... can't remember ;)

File: src/lang/en-US/index.json (hunk 2 above)
Comment (benhaynes) [reply src/lang/en-US/index.json:243]: OK!

File: src/layouts/tabular/tabular.vue (hunk 3 above)
Comment (benhaynes) [reply src/layouts/tabular/tabular.vue:85]: OK! Gotta figure out how to use the scoped translations....

**Pull request merged**